packages/client/test/rpc/engine/kaustinen6.spec.ts

```diff
@@ -80,6 +81,7 @@ describe(`valid verkle network setup`, async () => {
   const { server, chain, common } = await setupChain(kaustinen6Data, 'post-merge', {
     engine: true,
     genesisStateRoot: genesisVerkleStateRoot,
+    customCrypto: { verkleCrypto: verkle },
```

Comment:
Some nit here: can we call this `verkle` instead of `verkleCrypto`?

This is otherwise redundant, makes the API (a bit) less elegant and is also not in line with namings like `kzg` (which is also not named `kzgCrypto` e.g.).

Should be some relatively simple and quick search-and-replace task.

Reply:
Update: it would also simplify a lot of assignments like `customCrypto: { verkleCrypto: verkle }` to `customCrypto: { verkle }` further down the line.